### PR TITLE
Partial Upload

### PR DESCRIPTION
**Pull request by cliffng**
Result collection continues.

### CODE REVIEW

**Comment by DanishAlsayed** [comment]
I am very confused looking at this PR. For starters,
1- Please remove all the `.txt` files
2- Why did you replace everything in `main.py`? Please bring that back, if you want to add additional code please do so by adding it as functions in the appropriate files that we already have.
3- Why are there multiple files named like `run_result/main10y1stxcur.py`? Whats the `run_result` directory for? All those files seem to be very similar, can you please write your code in a generic way instead of copying and pasting all over the place? Why did you have to write these files anyway??

File: results.csv [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+Gradient Boosting,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
+FNN,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
+LSTM Regressor,,Date,10,"second order differencing, 80/20 ordered slice"
+AdaBoost Classifier,51.39%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+SVM,55.38%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+KNN,53.39%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+Random Forest Classifier,54.98%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+Logistic Regression,55.78%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+Gradient Boosting,55.38%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
+FNN,73.31%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
```
Comment (DanishAlsayed) [inline results.csv]: Are you sure about FNN getting up to 73%? Double check it.

File: results.csv [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+LSTM Regressor,,Date,10,"first order differencing, 80/20 ordered slice"
+Adaboost Classifier,59.76%,Date,10,"second order differencing, 80/20 ordered slice"
+SVM Classifier,55.53%,Date,10,"second order differencing, 80/20 ordered slice"
+KNN,55.13%,Date,10,"second order differencing, 80/20 ordered slice"
+Random Forest Classifier,53.72%,Date,10,"second order differencing, 80/20 ordered slice"
+Logistic Regression,55.13%,Date,10,"second order differencing, 80/20 ordered slice"
+Gradient Boosting,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
+FNN,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
+LSTM Regressor,,Date,10,"second order differencing, 80/20 ordered slice"
+AdaBoost Classifier,51.39%,"Cur, NASDAQ",5,"first order differencing, 80/20 ordered slice"
```
Comment (DanishAlsayed) [inline results.csv]: This looks funny, why would AdaClassifier suddenly fall to 51%? If you do not have an explanation then please verify it again.

File: results.csv [hunk 3]
```diff
@@ -1,16 +1,53 @@
...
+Gradient Boosting,56.34%,Date,10,"first order differencing, 80/20 ordered slice"
+FNN,55.94%,Date,10,"first order differencing, 80/20 ordered slice"
+LSTM Regressor,,Date,10,"first order differencing, 80/20 ordered slice"
+Adaboost Classifier,59.76%,Date,10,"second order differencing, 80/20 ordered slice"
+SVM Classifier,55.53%,Date,10,"second order differencing, 80/20 ordered slice"
+KNN,55.13%,Date,10,"second order differencing, 80/20 ordered slice"
+Random Forest Classifier,53.72%,Date,10,"second order differencing, 80/20 ordered slice"
+Logistic Regression,55.13%,Date,10,"second order differencing, 80/20 ordered slice"
+Gradient Boosting,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
+FNN,55.94%,Date,10,"second order differencing, 80/20 ordered slice"
```
Comment (DanishAlsayed) [inline results.csv]: So moving from 1st order to second order differencing made no difference on the FNN results?

File: results.csv (hunk 2 above)
Comment (DanishAlsayed) [inline results.csv]: In the results where you excluded the currencies & NASDAQ, did you add the `order` column we discussed? If not please do add it and retry, if you did already then add it in the comments section.

Comment (cliffng) [reply results.csv]: I thought you want to have that in a separate result set. I am going to run a separate set with out currencies and NASDAQ plus the order column.

File: results.csv (hunk 3 above)
Comment (cliffng) [reply results.csv]: This was the result obtained, but I will run a few times again and check if the model is not running properly

File: results.csv (hunk 2 above)
Comment (DanishAlsayed) [reply results.csv]: Okay, fine by me.

Comment (cliffng) [reply results.csv]: I can re-run this part too

File: results.csv (hunk 1 above)
Comment (cliffng) [reply results.csv]: As did not change the code except the skipping the removal of [Unnamed: 0] mentioned in whatsapp, which was found redundant. I will check this part

**Comment by DanishAlsayed** [comment]
The only changes you should have in this pull request should be in the `results.csv` as discussed yesterday.

File: results.csv (hunk 1 above)
Comment (cliffng) [reply results.csv]: Commit [e32eefd]

**Pull request closed without merging**